tasks/format_code.py

```diff
@@ -31,7 +31,6 @@ def format(ctx, check=False):
 
     flake8_cmd = [
         "python3 -m flake8",
-        "{}".format("--format" if not check else ""),
```

Comment:
The `--format` flag in `flake8` is not an opt-in flag to actually change the code (as `flake8` does not modify the sources), instead it is used to specify a different formatter to check for style warnings.